### ISSUE TITLE
Weather widget - use free API

### ISSUE THREAD

**Issue by LaxusCroco** [CrocoBlock/suggestions#6987]
Would it be possible to also allow users to use free weather API's like OpenWeatherMap in the weather widget in the future? Weatherbit is expensive.

**Comment by SolunaCG** [comment]
Yea I fully agree! Support for a free API would be great!

**Comment by SolunaCG** [comment]
Even though this is a slightly different topic but if #7023 could be fixed so that we can actually use the REST API with objects and not only with arrays then we could easily build our own weather "widget" with crocoblock! This would open up so many possibilities for even more dynamic websites (not only with weather but with every API)!

**Comment by OliverMannherz** [comment]
Yes, please make it work to work with other API keys like weatherapi.com. Thank you

**Comment by aalexandr21** [comment]
was added in JetElements 2.8.0

<img width="1086" height="431" alt="Image" src="https://github.com/user-attachments/assets/f9feeb82-8e29-4c82-8ac7-f2fbb3030732" />
